.travis.yml

```diff
@@ -13,4 +13,4 @@ script:
   - yarn test
   - pip install .
   - jupyter labextension install .
-  - python -m jupyterlab.browser_check
```

Comment:
I'd rather not remove the JupyterLab test. It's a good sanity test to ensure we are passing the JupyterLab tests and not breaking it any way (They'll keep expanding over time and we want to keep up with it, even if now we are tests are already covering that)